doc/classes/String.xml

```diff
@@ -599,6 +701,24 @@
 				Formats a string to be at least [code]min_length[/code] long by adding [code]character[/code]s to the right of the string.
 			</description>
 		</method>
+		<method name="rreplace" qualifiers="const">
```

Comment:
What's the difference with `replace`? The description seems to be the same.

Reply:
`rreplace` makes replacement starting from the right of the string. In a simple example:
```gdscript
"a a".replace("a", "B", 1) # prints "B a"
"a a".rreplace("a", "B", 1) # prints "a B"
```
In the above example, these calls are equivalents of `String.replace_first` and `String.replace_last` functionality-wise and I realized this after updating those functions. Will open a proposal noting this, and to discuss whether we want to coalesce these.